test/e2e/delete_test.go

```diff
@@ -18,28 +18,28 @@ func deletionTestSuite(t *testing.T) {
 
 // testWindowsNodeDeletion tests the Windows node deletion from the cluster.
 func testWindowsNodeDeletion(t *testing.T, nodeCount int) {
-	testCtx := framework.NewTestCtx(t)
-	namespace, err := testCtx.GetNamespace()
+	testCtx := NewTestContext(t, nodeCount)
+	namespace, err := testCtx.osdkTestCtx.GetNamespace()
 	if err != nil {
 		t.Fatal(err)
 	}
-
+	testCtx.numberOfNodes = 0
```

Comment:
We don't need this any more.

Reply:
I am assuming you will remove it then as part of this PR